yoti-sdk-sandbox/suppressed-cves.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
+   <suppress>
+      <notes><![CDATA[
+        The problem is described here: https://nvd.nist.gov/vuln/detail/CVE-2015-5237
+        Google maintain it's fixed from 3.4 onwards: https://github.com/google/protobuf/issues/760
+      ]]></notes>
+      <gav>com.google.protobuf:protobuf-java:3.5.0</gav>
+      <cve>CVE-2015-5237</cve>
+   </suppress>
+   
+   <suppress>
+      <notes><![CDATA[
+        The problem is described here: https://nvd.nist.gov/vuln/detail/CVE-2017-17485#VulnChangeHistoryDiv
```

Comment:
Can these be resolved if we upgrade Jackson and remove support for 1.6?

Reply:
I think so, it's ok to remove the support of the Java version 1.6 then?

Reply:
I think it's ok to remove it.

Reply:
Can do.  I have already merged these CVE exclusions though